src/main/webapp/components/table/CustomTableControl.jsx

```diff
@@ -5,6 +5,14 @@ import { _ } from '@splunk/ui-utils/i18n';
 import { getUnifiedConfigs } from '../../util/util';
 import { getBuildDirPath } from '../../util/script';
 
+function onCustomControlError(params) {
+    const docURL = `https://splunk.github.io/addonfactory-ucc-generator/custom_ui_extensions/custom_hook/`;
```

Comment:
link should be this one 
`https://splunk.github.io/addonfactory-ucc-generator/custom_ui_extensions/custom_control/` 
something to think of is to create some const/map with all docs links and just choose specific one from there, 
just to keep it outside and unified and also reuse function as that one is actually almost the same as `onCustomHookError`